lib/pagem.rb

```diff
@@ -58,11 +58,11 @@ def render(opt = {})
       href = "##{@page_variable}"
 
       content_tag('div',
-        (icon_link('/images/pagem/arrow_leftend.gif', I18n.t('application.pagination.first', :default => "First"), href, link_options(1, p > 1)) +
-      icon_link('/images/pagem/arrow_left.gif', I18n.t('application.pagination.previous', :default => "Previous"), href, link_options(p - 1, p > 1)) +
+        (icon_link('/assets/pagem/arrow_leftend.gif', I18n.t('application.pagination.first', :default => "First"), href, link_options(1, p > 1)) +
+      icon_link('/assets/pagem/arrow_left.gif', I18n.t('application.pagination.previous', :default => "Previous"), href, link_options(p - 1, p > 1)) +
       pager_section(p, tp) +
-      icon_link('/images/pagem/arrow_right.gif', I18n.t('application.pagination.next', :default => "Next"), href, link_options(p + 1, p < tp, true)) +
-      icon_link('/images/pagem/arrow_rightend.gif', I18n.t('application.pagination.last', :default => "Last"), href, link_options(tp, p < tp, true))) +
+      icon_link('/assets/pagem/arrow_right.gif', I18n.t('application.pagination.next', :default => "Next"), href, link_options(p + 1, p < tp, true)) +
+      icon_link('/assets/pagem/arrow_rightend.gif', I18n.t('application.pagination.last', :default => "Last"), href, link_options(tp, p < tp, true))) +
```

Comment:
Does the /images on line 20 also need to be updated? Also, instead of using the /assets/ path directly, can we use ```"#{image_path('pagem/arrow_rightend.gif')}``` to correctly generate the path (whether or not the user is using the asset pipeline)?